oqs/oqs.py

```diff
@@ -37,13 +45,48 @@ def _load_shared_obj(name):
     raise RuntimeError("No " + name + " shared libraries found")
 
 
+def _install_liboqs(directory):
+    """Attempts to install liboqs automatically."""
+    oqs_install_str = (
+        "cd "
+        + directory
+        + """
+git clone https://github.com/open-quantum-safe/liboqs --depth 1
```

Comment:
May it be better to tie this down to a specific `liboqs` release (e.g., 0.10.0) and not to "main"?